src/core_modules/capture-core/components/Pages/Search/SearchForm/SearchForm.epics.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import { catchError, flatMap, map, startWith } from 'rxjs/operators';
+import { from } from 'rxjs/observable/from';
+import { of } from 'rxjs/observable/of';
+import { searchPageActionTypes } from '../SearchPage.container';
+import { getTrackedEntityInstances } from '../../../../trackedEntityInstances/trackedEntityInstanceRequests';
+import {
+    getTrackedEntityTypeThrowIfNotFound,
+    getTrackerProgramThrowIfNotFound,
+} from '../../../../metaData';
+import { actionCreator } from '../../../../actions/actions.utils';
+
+const trackerCaptureAppUrl = () => (process.env.REACT_APP_TRACKER_CAPTURE_APP_PATH || '..').replace(/\/$/, '');
+
+export const onScopeProgramFindUsingUniqueIdentifierEpic = (action$: InputObservable, store: ReduxStore) =>
```

Comment:
I have created here two epics. One for the lookup where the scope is `TrackedEntityType` and one for when the scope is simply a `Program`.